test/Test/Common.hs

```diff
@@ -68,6 +68,7 @@ originateBaseDaoWithConfig contractExtra config = do
         ]
     )
 
+-- TODO: [#96] Add a way to customize storage
 originateBaseDaoWithBalance
```

Comment:
@pasqu4le I've made the change as you suggested, also added an issue as well: https://github.com/tqtezos/baseDAO/issues/96

You can check this to see if it fits what you have in mind. In the meanwhile, the delphinet tests probably take 1 or 2 hours to finish.

Reply:
LGTM